apis/r/DESCRIPTION

```diff
@@ -43,7 +43,8 @@ Imports:
     arch (>= 0.0.2),
     dplyr,
     data.table,
-    spdl
+    spdl,
+    rlang
```

Comment:
This is missing the matching use in `NAMESPACE`:

```r
N  checking dependencies in R code (1.7s)
   Namespace in Imports field not imported from: ‘rlang’
     All declared Imports should be used.
```

Reply:
Fixed in fb87cd2

Reply:
Dependency on rlang removed in e35142e